workflow/rules/common.smk

```diff
@@ -31,11 +31,15 @@ def get_compressed_docs(wildcards):
 def get_fhir_metadata(wildcards):
     return pep.sample_table.loc[wildcards.id][["fhir_metadata"]]
 
+def get_additional_metadata(wildcards):
+    if type(pep.sample_table.loc[wildcards.id]["additional_metadata"]) == str:
```

Comment:
```suggestion
   if "additional_metadata" in pep.sample_table.columns:
```

I think pandas does not recognize the last column if it's empty. No clue why that is. Alternatively, we can check if "additional_metadata" is in the column names